### ISSUE TITLE
FAB shadow is rendered wrongly on API 15

### ISSUE THREAD

**Issue by akwizgran** [leinardi/FloatingActionButtonSpeedDial#93]
On the Sony Xperia Tipo (API 15) the shadow behind the FAB isn't properly aligned with the button - a white line is visible between the button and its shadow at the bottom left and right corners.

**Comment by akwizgran** [comment]
![device-2018-09-27-103254.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5baca32735dc3600131efd31/125516c66e82c728ace21e0d46ec918c2a8186e4ddb9ee1be90da9d97b116a227957f17e24c7af6aaf50740b0a0c0abe4b4a5a15aff1da28a5e73f79895d5eebd16d5e26c46b44a4bd8a0da313f871d51bdaebe5c01982fa3737e58a854e3b1d0d188dff5a3ca29e43cd4abb.png)

**Comment by leinardi** [comment]
This is a bug of the `FloatingActionButton` from the Design library. I have open an issue, feel free to star it: https://issuetracker.google.com/issues/77303906

Related to #11

**Comment by akwizgran** [comment]
We have another FAB in the app where this doesn't seem to happen. Any thoughts about what might be different between the two FABs?

![device-2018-09-27-131342](https://user-images.githubusercontent.com/1458432/46145176-390e9380-c257-11e8-904b-ca94de8246a0.png)

**Comment by leinardi** [comment]
Yep, just try to change the size of the FAB programmatically (define it mini in XML and than set it to normal in Java/Kotlin:

```
 // fab has app:fabSize attribute set to "mini"
 FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab2);
 fab.setSize(FloatingActionButton.SIZE_NORMAL);
```
